servant/src/Servant/API/Description.hs

```diff
@@ -1,52 +1,51 @@
-{-# LANGUAGE DataKinds           #-}
-{-# LANGUAGE DeriveDataTypeable  #-}
-{-# LANGUAGE FlexibleContexts    #-}
-{-# LANGUAGE PolyKinds           #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE PolyKinds #-}
 {-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TypeFamilies        #-}
-{-# LANGUAGE TypeOperators       #-}
-{-# OPTIONS_HADDOCK not-home    #-}
-module Servant.API.Description (
-    -- * Combinators
-    Description,
-    Summary,
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# OPTIONS_HADDOCK not-home #-}
+
+module Servant.API.Description
+  ( -- * Combinators
+    Description
+  , Summary
+
     -- * Used as modifiers
-    FoldDescription,
-    FoldDescription',
-    reflectDescription,
-    ) where
+  , FoldDescription
+  , FoldDescription'
+  , reflectDescription
+  )
+where
 
-import           Data.Kind
-                 (Type)
-import           Data.Proxy
-                 (Proxy (..))
-import           Data.Typeable
-                 (Typeable)
-import           GHC.TypeLits
-                 (KnownSymbol, Symbol, symbolVal)
+import Data.Kind (Type)
+import Data.Proxy (Proxy (..))
+import Data.Typeable (Typeable)
+import GHC.TypeLits (KnownSymbol, Symbol, symbolVal)
 
 -- | Add a short summary for (part of) API.
 --
 -- Example:
 --
 -- >>> type MyApi = Summary "Get book by ISBN." :> "books" :> Capture "isbn" Text :> Get '[JSON] Book
 data Summary (sym :: Symbol)
-    deriving (Typeable)
+  deriving (Typeable)
 
 -- | Add more verbose description for (part of) API.
 --
 -- Example:
 --
 -- >>> :{
---type MyApi = Description
+-- type MyApi = Description
 --  "This comment is visible in multiple Servant interpretations \
 --  \and can be really long if necessary. \
 --  \Haskell multiline String support is not perfect \
 --  \but it's still very readable."
 -- :> Get '[JSON] Book
```

Comment:
I suspect the indentation here should be increased, wrt https://github.com/haskell-servant/servant/actions/runs/15779239616/job/44480594133?pr=1825